uikit/views/src/main/java/com/sygic/maps/uikit/views/poidetail/viewmodel/PoiDetailInternalViewModel.kt

```diff
@@ -39,16 +39,15 @@ private val SHOWCASE_DELAY = TimeUnit.SECONDS.toMillis(1)
 const val DEFAULT_BEHAVIOR_STATE = BottomSheetBehavior.STATE_COLLAPSED
 const val SHOWCASE_BEHAVIOR_STATE = BottomSheetBehavior.STATE_EXPANDED
 
-internal class PoiDetailInternalViewModel(poiDetailData: PoiDetailData,
-                                          private val preferencesManager: PreferencesManager
-) : ViewModel(), BottomSheetBehaviorWrapper.StateListener {
+internal class PoiDetailInternalViewModel(private val preferencesManager: PreferencesManager) : ViewModel(),
+    BottomSheetBehaviorWrapper.StateListener {
 
-    val titleText: String = poiDetailData.titleString
-    val subtitleText: String = poiDetailData.subtitleString
-    val urlText: String? = poiDetailData.urlString
-    val emailText: String? = poiDetailData.emailString
-    val phoneText: String? = poiDetailData.phoneString
-    val coordinatesText: String? = poiDetailData.coordinatesString
+    val titleText: MutableLiveData<String> = MutableLiveData()
```

Comment:
```suggestion
    val titleText: LiveData<String> = MutableLiveData()
```
hmm, toto nie je moc ideal... teraz si z imutable veci spravil mutable. Minimalne to definuje ako `LiveData`